docs/integration/laravel.md

```diff
@@ -27,15 +27,15 @@ key will not work.
 ```diff
 # config/filesystems.php
 
-    'dynamodb' => [
--       'driver' => 'dynamodb',
-+       'driver' => 'async-aws-dynamodb',
-        'key' => env('AWS_ACCESS_KEY_ID'),
-        'secret' => env('AWS_SECRET_ACCESS_KEY'),
-        'region' => env('AWS_DEFAULT_REGION', 'us-east-1'),
-        'table' => env('DYNAMODB_CACHE_TABLE', 'cache'),
-        'endpoint' => env('DYNAMODB_ENDPOINT'),
-    ],
+'dynamodb' => [
```

Comment:
missing space  to avoid breaking `+`, `-` indentation

Reply:
Actually no. It surprised me too. 

<img width="865" alt="Screenshot 2020-05-10 at 11 39 13" src="https://user-images.githubusercontent.com/1275206/81495764-ef02e600-92b2-11ea-881a-c6c543cc82e5.png">

Reply:
the `'` of `'dynamodb` should not be aligned with `-` . Same for `],`
when indenting diff, all lignes are prefixed with `-`, `+` or ` `

Reply:
example of issue
```diff
-'foo' => null,
+'Foo' => null,
'dynamodb' => [
-   'driver' => 'dynamodb',
+   'driver' => 'async-aws-dynamodb',
    'key' => env('AWS_ACCESS_KEY_ID'),
],
'Bar' => null,
-'baz' => null,
+'Baz' => null,

Reply:
Ah, Okey. I understand now. I've fixed the diff and I've also modified other diffs with the same issue